Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+
+def get_commit_tag() {
+  return sh(returnStdout: true, script: "git log -n 1 --pretty=format:'%h'").trim()
+}
+
+node {
+  stage('Prep') {
+    checkout scm
+    tag = get_commit_tag()
+  }
+
+  stage ('Buld Image'){
+    python_app = docker.build('mdnwebdocs/developer-portal-app:${tag}', '-f ./dockerfiles/Dockerfile.app .')
+    node_app = docker.build('mdnwebdocs/developer-portal-static:${tag}', '-f ./dockerfiles/Dockerfile.static .')
+  }
+
+  stage('Push Image') {
```

Comment:
Before we push the images, I think we should ensure that these tests pass:
```
docker-compose exec app python manage.py test
docker-compose exec static npm run test
```
but perhaps you were planning to do that later in a separate PR.

Reply:
I was planning a separate PR for this. The docker-compose file will need to be modified to take the new images so lets make sure it gets built and pushed first